### PR TITLE
MRG: Faster raw plotting with decim

### PR DESCRIPTION
**Pull request by Eric89GXL**
This should speed up our raw plotting when low-passing is done.

The same thing can be done with Epochs once this is tested / works.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
cc @jaeilepp

File: mne/viz/raw.py [hunk 1]
```diff
@@ -194,6 +194,14 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
         modes are overrided with ``order`` keyword.
     butterfly : bool
         Whether to start in butterfly mode. Defaults to False.
+    decim : int | 'auto'
+        Amount to decimate the data during display for speed purposes.
+        You should only decimate if the data are sufficiently low-passed,
+        otherwise aliasing can occur. The 'auto' mode (default) uses
+        the decimation that results in a Nyquist frequency at least 2 times
+        larger than ``min(info['lowpass'], lowpass)`` (e.g., a 40 Hz lowpass
+        will result in at least a 80 Hz Nyquist frequency and thus 160 Hz
```
Comment (agramfort) [inline mne/viz/raw.py]: 120

**Comment by Eric89GXL** [comment]
Done

File: mne/viz/raw.py [hunk 2]
```diff
@@ -194,6 +194,14 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
...
     butterfly : bool
         Whether to start in butterfly mode. Defaults to False.
+    decim : int | 'auto'
+        Amount to decimate the data during display for speed purposes.
+        You should only decimate if the data are sufficiently low-passed,
+        otherwise aliasing can occur. The 'auto' mode (default) uses
+        the decimation that results in a Nyquist frequency at least 1.5 times
+        larger than ``min(info['lowpass'], lowpass)`` (e.g., a 40 Hz lowpass
+        will result in at least a 60 Hz Nyquist frequency and thus 120 Hz
+        displayed sample rate).
```
Comment (agramfort) [inline mne/viz/raw.py]: @Eric89GXL can you simplify this sentence as we discussed. Basically a lowpass of 40Hz results results to a decimation up to a sampling rate of 120Hz.

Comment (Eric89GXL) [inline mne/viz/raw.py]: done

**Comment by agramfort** [comment]
thx @Eric89GXL 

@jaeilepp it will be great if you can do this with Epochs.plot now

**Pull request merged**